controllers/storagerequest/storagerequest_controller.go

```diff
@@ -213,7 +213,7 @@ func (r *StorageRequestReconciler) initPhase() error {
 	}
 
 	// check request status already contains the name of the resource. if not, add it.
-	if r.StorageRequest.Spec.Type == "blockpool" {
+	if strings.ToLower(r.StorageRequest.Spec.Type) == "block" {
```

Comment:
You are repeating string manipulation logic in multiple places. Instead of codifying the implementation in each code site use a util func to codify the intent, so if the implementation changes we will only need to change it once

```suggestion
	if isBlockRequest(r.StorageRequest.Spec.Type) {
```

Reply:
done.